wagtail/admin/side_panels.py

```diff
@@ -68,6 +68,7 @@ def get_context_data(self, parent_context):
                         .select_related("locale")
                         if user_perms.for_page(translation).can_edit()
                     ],
+                    "translations_total": self.page.get_translations().count() + 1,
```

Comment:
suggestion: since this now looks at locales, perhaps this variable should reflect that. 

-OR-

a comment as to why +1

Reply:
I've went with the comment option 😀, only because it feels more consistent to to me to have "translations_total" from "get_translations.count".